apps/cli/src/lib/compiler/comamnds.ts

```diff
@@ -0,0 +1,57 @@
+import glob from "glob";
+import { readFile, writeFile } from "node:fs/promises";
+import path from "node:path";
+import { CompilerAssets } from "./assets";
+
+export async function parseCommands(workbench: string) {
+  const commandsDir = path.join(workbench, "commands");
+
+  const commandsFiles = await new Promise<string[]>((resolve, reject) => {
+    glob(`${commandsDir}/**/*.js`, (err, files) => {
+      if (err) {
+        reject(err);
+      } else {
+        resolve(files);
+      }
+    });
+  });
+
+  const commands = await Promise.all(
+    commandsFiles.map(async (file) => {
+      let code = await readFile(file, "utf-8");
+      const commandName = path.basename(file, ".js");
+
+      code = code.replace(
+        /export default class ([a-zA-Z0-9]+) /,
+        `export class ${commandName} `
+      );
+
+      return await writeFile(file, code);
```

Comment:
A bit confused here, doesn't `writeFile` return `Promise<void>`? Would that make `commands` an array of `void`?

Reply:
[a93e621](https://github.com/Disploy/disploy/pull/19/commits/a93e621c227e2f318196fdec011c7abc89f8b4ec)